docs/dev/engine_overview.rst

```diff
@@ -311,3 +311,88 @@ the parameter ``template`` must be set to the desired type.
    address.postcode          postcode of object
    address.country           country of object
    ========================= =====================================================
+
+.. _BibTeX format: https://www.bibtex.com/g/bibtex-format/
+.. _BibTeX field types: https://en.wikipedia.org/wiki/BibTeX#Field_types
+
+.. list-table:: Parameter of the **paper** media type /
+                see `BibTeX field types`_ and `BibTeX format`_
+   :header-rows: 2
+   :width: 100%
+
+   * - result-parameter
+     - Python type
+     - information
+
+   * - template
+     - :py:class:`str`
+     - is set to ``paper.html``
+
+   * - title
+     - :py:class:`str`
+     - title of the result
+
+   * - content
+     - :py:class:`str`
+     - abstract
+
+   * - comments
+     - :py:class:`str`
+     - free text display in italic below the content
+
+   * - tags
+     - :py:class:`List <list>`\ [\ :py:class:`str`\ ]
+     - free tag list
+
+   * - publishedDate
+     - :py:class:`datetime <datetime.datetime>`
+     - last publication date
+
+   * - authors
+     - :py:class:`List <list>`\ [\ :py:class:`str`\ ]
+     - list of authors of the work (authors with a "s")
+
+   * - editor
+     - :py:class:`str`
+     - list of editors of a book
+
+   * - publisher
+     - :py:class:`str`
+     - name of the publisher
+
```

Comment:
I am just in translation of the strings .. and I miss a description of the [`result.type` field](https://github.com/searxng/searxng/blob/master/searx/templates/simple/result_templates/paper.html#L24) .. is the field intended to characterize the medium (e.g. pdf, book, paper)?

Reply:
It sounds like what you have described.

It is mix between the file format and "entry type" (in the BibTex vocabulary):
* springer: I can't remember what the [`contentType` field](https://github.com/searxng/searxng/blob/bafebe91ac9812c4ac243b0717f411b1ed0a66a9/searx/engines/springer.py#L55) is. It sounds like the file format.
* crossref: `book-chapter`, `book`, `other`, `component`
* google schoolar: `book` but also `pdf`, `html`. This is the word before the title. In the following example, it is "book":
![image](https://user-images.githubusercontent.com/1594191/192087254-48f79e12-8182-4f18-9a90-96a9759d38d6.png)